src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/customer.html.twig

```diff
@@ -85,7 +96,7 @@
         <div class="tab-content">
           <div class="tab-pane fade active show" id="customer-carts-tab-content" role="tabpanel" aria-labelledby="pills-home-tab">
             <table class="table" id="customer-carts-table">
-              <thead>
+              <thead class="d-none">
```

Comment:
MM, it's weird to pu this class on a tag that is not identified, besides I couldn't find any reference to `customer-carts-table` or `customer-carts-tab-content` How exactly do you toggle this class to display it?